src/pages/debrief/components/vehicle-checks-card/__mocks__/vehicle-checks-card.mock.ts

```diff
@@ -0,0 +1,42 @@
+import { QuestionResult } from '@dvsa/mes-test-schema/categories/common';
+
+export class VehicleChecksCardMock {
+
+  private static instance: VehicleChecksCardMock;
+
+  private constructor() { }
+
+  public static getInstance(): VehicleChecksCardMock {
+    if (!VehicleChecksCardMock.instance) {
+      VehicleChecksCardMock.instance = new VehicleChecksCardMock();
+    }
+    return VehicleChecksCardMock.instance;
+  }
+
+  public getMalformedVehicleChecks(): QuestionResult[] {
+    return [
+      {
+        code: 'T1',
+        description: 'Safety factors while loading',
+      },
+      {
+        code: 'T2',
+        description: 'Reflectors condition',
+        outcome: 'DF',
+      },
+      {
+        code: 'S1',
+        description: 'All doors secure',
+      },
+      {
+        code: 'S2',
+        description: 'Air leaks',
+        outcome: 'P',
+      },
+      {
+        code: 'S4',
+        description: 'Mudguards condition',
+      },
+    ];
+  }
+}
```

Comment:
Doesn't need a singleton for this, or indeed a class. Just export the function getMalformedVehicleChecks and import and call that in the spec file.